crates/node_runtime/src/node_runtime.rs

```diff
@@ -206,6 +206,10 @@ impl RealNodeRuntime {
 
         anyhow::Ok(node_dir)
     }
+
+    // pub fn configure_proxy(&self) {
```

Comment:
Do we need this?

Reply:
Sorry for lefting debug function here. I'm trying to make `npm install` working through the proxy.